languages/csharp/reference/exercise-concepts/grade-school.md

```diff
@@ -0,0 +1,25 @@
+# Concepts of grade-school
+
+[Example implementation](https://exercism.io/tracks/csharp/exercises/grade-school/solutions/2fd52e1bbfcecdfaff4075c4)
+
+## General
+
+- functions: used as the main entry point for the exercise
+- function arguments: input strand is passed as an argument
+- return values: returning a value from a method
+- scoping: use `{` and `}` to denote scoping
+- interfaces: the type returned is an `IEnumerable<string>`
+- classes: the tested method is defined in a class
+- objects: creating an instance of the `SortedList<string, int>` class
+- state: keeping track of the grades
+- static: the tested method is a static method
```

Comment:
This one isn't static since you have to keep track of `_students`

Reply:
Correct! Removed it.